### DISCUSSION TITLE
using markdown, is there a way to re-use a chunk of text in multiple places?

### DISCUSSION THREAD

**Discussion by mattgwiz** [nuxt/nuxt#19250] [Questions]
# Reusing text blocks
- Is there a way using markdown to save a text-block somewhere, and then insert it wherever needed without having to copy/paste? 
- The use-case for this is to create it in one place, save it in one place, and change it in one place, and not have to hunt for everywhere it occurs in documents in the event the content does change.

**Comment by ndragun92** [comment]
[Nuxt 3 content](https://content.nuxtjs.org/get-started)

This module and creating .md files?

**Reply by mattgwiz** [reply]
not seeing anything that helps, sorry.
